### ISSUE TITLE
FindOneOptions refers to  incorrect class 

### ISSUE THREAD

**Issue by booradleys** [typeorm/typeorm#579]
FindOneOptions refers to  :
node_modules\typeorm\driver\mongodb\typings.d.ts 
instead of:
C:\Applications\Node\exotica\server\node_modules\typeorm\find-options\FindOneOptions.d.ts

So my application doesn't compile anymore.
Can anyone help me ?

**Comment by pleerock** [comment]
fixed in `0.1.0-alpha.21`

**Comment by booradleys** [comment]
Thank you,
But now i get:

Module '"node_modules/typeorm/index"' has no exported member 'FindOneOptions'.

````
import { FindOneOptions, FindManyOptions } from "typeorm"; 
````
